src/api/wix/WixToolset.Data/ErrorMessages.cs

```diff
@@ -2266,6 +2266,16 @@ public static Message IllegalAttributeWhenNested(SourceLineNumber sourceLineNumb
             return Message(sourceLineNumbers, Ids.IllegalAttributeWhenNested, "The File element contains an attribute '{0}' that cannot be used in a File element that is a child of a Component element.", attributeName);
         }
 
+        public static Message OverlengthTableNameInMergeModule(SourceLineNumber sourceLineNumbers, string tableName)
```

Comment:
This message isn't necessary; the second one will suffice (and I prefer its message too).